index.html

```diff
@@ -44,5 +44,7 @@
         <!-- MoodoMovie References  -->
         <script src="assets/javascript/text-analysis.js"></script>
         <script src="assets/javascript/app.js"></script>
```

Comment:
I think app.js should be the last reference inserted, once it's the main code for the whole application, and it should be the start point.